File: scripts/fix-video.py
with open("src/lib/seo/video-index-page.tsx", "r", encoding="utf-8") as f:
    text = f.read()

good = """export function siteVideoIndexMetadata(site: Site): Metadata {
  if (!site) return { title: 'Not found' };
  const base = siteOrigin(site) || 'https://letsgetquoted.com';
  
  const entries = getAllPublishedVideos(site.content);"""

import re
text = re.sub(r'export function siteVideoIndexMetadata[\s\S]*?const entries = getAllPublishedVideos\(site\.content\);', good, text)
with open("src/lib/seo/video-index-page.tsx", "w", encoding="utf-8") as f:
    f.write(text)
print("Fixed via regex")

File: scripts/fix-video.js
const fs = require('fs');
let code = fs.readFileSync('src/lib/seo/video-index-page.tsx', 'utf-8');

code = code.replace(
  `export function siteVideoIndexMetadata(site: Site): Metadata {

  if (!site)   const base = site ? siteOrigin(site) || 'https://letsgetquoted.com' : 'https://letsgetquoted.com';
  return {
    alternates: { canonical: \`\${base}/videos\` }, title: 'Not found' };`,
  `export function siteVideoIndexMetadata(site: Site): Metadata {
  if (!site) return { title: 'Not found' };
  const base = siteOrigin(site) || 'https://letsgetquoted.com';`
);

fs.writeFileSync('src/lib/seo/video-index-page.tsx', code);
